airbyte-integrations/bases/base-normalization/normalization/transform_catalog/transform.py

```diff
@@ -145,8 +145,12 @@ def read_json_catalog(input_path: str) -> dict:
     return json.loads(contents)
 
 
+def is_union_type(property_type) -> bool:
+    return type(property_type) is list and len(property_type) > 2
```

Comment:
What if we get a union type non-nullable? 
`"type": ["integer", "string"]`

Reply:
good point, will change to handle non-nullables